recipes/master.rb

```diff
@@ -0,0 +1,26 @@
+#
+# Cookbook Name:: osl-jenkins
+# Recipe:: master
+#
+# Copyright 2014, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+node.default['jenkins']['master']['install_method'] = 'war'
+node.default['jenkins']['master']['version'] = '1.557'
+node.default['jenkins']['master']['source'] = 'http://mirrors.jenkins-ci.org' \
+  "/war/#{node['jenkins']['master']['version']}/jenkins.war"
+node.default['jenkins']['master']['checksum'] = '4507a49529d15985562dfa50ca' \
```

Comment:
Would it make more sense to just have this hash all on one line below? Or is that against the style code?